coderd/database/dbfake/dbfake.go

```diff
@@ -59,15 +59,15 @@ func WorkspaceWithAgent(
 		agents = m(agents)
 	}
 	ws := Workspace(t, db, seed)
-	WorkspaceBuild(t, db, ws, database.WorkspaceBuild{}, &sdkproto.Resource{
+	NewWorkspaceBuildBuilder(t, db, ws).Resource(&sdkproto.Resource{
```

Comment:
Could we do `NewWorkspaceBuilder` instead? Then the `Build` function would implicitly do what we expect as well.

Reply:
I considered the same, then I wondered what we would call a "Builder" for a workspace, if we ever need one.. 😅

Reply:
The pattern I'm planning to follow is `NewXxxxxBuilder()` for the function that creates a builder (where Xxxxx is the resource type), so `NewWorkspaceBuildBuilder()` is the builder for a `WorkspaceBuild`. 

I'd eventually like to convert everything in `dbfake` to a builder pattern, and then I think we could potentially drop the `Builder` suffix and the `New` prefix for the sake of readability.  But while I'm changing things over, it's useful to be able tell what is and isn't a builder.